aims/aim1.md

```diff
@@ -0,0 +1,44 @@
+# Aim 1:
+
+**Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data.**
+
+## Background
+
+Deep unsupervised models can generate hypothetical data by learning and decoding a lower dimensional latent space.
+The latent space of such models, including Restricted Boltzmann Machines (RBMs), Variational Autoencoders (VAEs), and Generative Adversarial Networks (GANs), preserves non-linear distances between data points and represents features that drive variation in the input data [1,2,3].
+Thus, the algorithms serve a dual purpose: (1) non-linear dimensionality reduction and (2) the ability to interpolate between data.
+Applied to image data, the algorithms, including the recurrent VAE DRAW network, have shown much promise in generating realistic looking fake data through the learning of a lower dimensional representation of different pixel arrangements [4]. 
+Additionally, GANs trained on fluorescent microscopy images of yeast cells enable interpolation of the learned latent space to model cell-cycle protein localization [5].
+As applied to single cell gene expression data, we hypothesize that such algorithms can learn complex biological features that enable cell-type segmentation and cell state interpolation.
+Previously, we have demonstrated the ability to train a VAE on bulk gene expression data to identify latent biological features [6].
+Therefore, the _objective of this aim_ is to train a deep neural network generative model built from single cell gene expression from the HCA demonstrating a proof of concept modeling approach.
+
+## Approach
+
+We will train a VAE on single cell transcriptome data from the HCA.
+In order to select the best model to validate, we will perform an extensive grid search with a held-out validation set to identify optimal VAE architecture and hyperparameters.
+We will select optimal models based on minimizing reconstruction loss plus a KL divergence term which constrains the features to a Gaussian distribution.
+
+In addition, we will compare our optimized VAE to an alternative VAE we will train with an augmentation approach.
+A common image processing goal is to disentangle invariant features from a corpus of images.
```

Comment:
I still think you might need a sentence that provides the intuition behind invariant features. I'm also looking for the _why_ here.